### PR TITLE
0.1/596 cc family sheet design individual family

### PR DESCRIPTION
**Pull request by alexalala**
individual sheet 

https://trello.com/c/3exJIJFy/596-cc-family-sheet-design-individual-family

:octocat:

### CODE REVIEW

File: public/family.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+				<ul>
+					<li><strong>1</strong> Child under 1</li>
+					<li><strong>1</strong> pregnancy</li>
+				</ul>
+			</div>
+			<table>
+				<tr>
+					<th>Collection Date</th>
+					<th>Amount of vouchers given</th>
+					<th>Participant Signature</th>
```
Comment (charlesstrange2) [inline public/family.html]: Change to "Collector Signature"

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
Change the Collector / Participant thing.

**Comment by alexalala** [comment]
@charlesstrange2 all ready now, did something dumdum though and committed while on my vagrant box, do you want me to rewind the commits and re commit it all?

**Review by charlesstrange2 — APPROVED** [review]
Looks OK too!
:christmas_tree:

**Pull request merged**